README.md

```diff
@@ -165,6 +166,12 @@ For example, a second release in June 2021 would be 202106.01. Although the SDK
 
 All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.1.2](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.2).
 
+### 202012.00
+
+[API documentation of 202012.00 release](https://docs.aws.amazon.com/embedded-csdk/202012.00/lib-ref/index.html)
+
+This release includes AWS IoT Over-the-air Update(Release Candidate), backoffAlgorithm, and PKCS11 libraries. Additionally, there is a major update to the coreJSON and coreHTTP APIs. All libraries continue to undergo code quality checks (e.g. MISRA-C compliance), and Coverity static analysis. In addition, all libraries except AWS IoT Over-the-air Update and backoffAlgorithm undergo validation of memory safety with the C Bounded Model Checker (CBMC) automated reasoning tool.
```

Comment:
```suggestion
This release includes AWS IoT Over-the-air Update(Release Candidate), backoffAlgorithm, and corePKCS11 libraries. Additionally, there is a major update to the coreJSON and coreHTTP APIs. All libraries continue to undergo code quality checks (e.g. MISRA-C compliance), and Coverity static analysis. In addition, all libraries except AWS IoT Over-the-air Update and backoffAlgorithm undergo validation of memory safety with the C Bounded Model Checker (CBMC) automated reasoning tool.
```

Reply:
core is not mentioned as part of the library names in this section. Please refer to the older releases. It is only mentioning MQTT, JSON and HTTP client library

Reply:
Hmm, okay. I suggested it because the second statement in the section is mentioned `coreMQTT` and `coreJSON` libraries. If this is the way prior sections have switched between generic and library brand naming, then I'm fine with it.